nano/node/ipc.cpp

```diff
@@ -398,7 +398,7 @@ node (node_a), rpc (rpc_a)
 
 nano::ipc::ipc_server::~ipc_server ()
 {
-	node.logger.always_log ("IPC: server stopped");
```

Comment:
should we check that `stop ()` hasn't already been called in cases like this?:
```
{
ipc_server server;
server.stop ();
}
```
Could cause issues I guess.